src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -0,0 +1,218 @@
+     *     structure as the rest of the controllers of this class.
+     *   * 'admin_index' is a nice shortcut to the backend homepage. This allows
+     *     to create simpler links in the templates. Moreover, in the future we
+     *     could move this annotation to any other controller while maintaining
+     *     the route name and therefore, without breaking any existing link.
+     *
+     * @Route("/", name="admin_index")
+     * @Route("/", name="admin_post_index")
+     * @Method("GET")
+     */
+    public function indexAction()
+    {
+        $entityManager = $this->getDoctrine()->getManager();
+        $posts = $entityManager->getRepository('AppBundle:Post')->findAll();
+
+        return $this->render('admin/blog/index.html.twig', array('posts' => $posts));
+    }
+
+    /**
+     * Creates a new Post entity.
+     *
+     * @Route("/new", name="admin_post_new")
+     * @Method({"GET", "POST"})
+     *
+     * NOTE: the Method annotation is optional, but it's a recommended practice
+     * to constraint the HTTP methods each controller responds to (by default
+     * it responds to all methods).
+     */
+    public function newAction(Request $request)
+    {
+        $post = new Post();
+        $post->setAuthorEmail($this->getUser()->getEmail());
+
+        // See http://symfony.com/doc/current/book/forms.html#submitting-forms-with-multiple-buttons
+        $form = $this->createForm('AppBundle\Form\PostType', $post)
+            ->add('saveAndCreateNew', 'Symfony\Component\Form\Extension\Core\Type\SubmitType');
+
+        $form->handleRequest($request);
+
+        // the isSubmitted() method is completely optional because the other
+        // isValid() method already checks whether the form is submitted.
+        // However, we explicitly add it to improve code readability.
+        // See http://symfony.com/doc/current/best_practices/forms.html#handling-form-submits
+        if ($form->isSubmitted() && $form->isValid()) {
+            $post->setSlug($this->get('slugger')->slugify($post->getTitle()));
+
+            $entityManager = $this->getDoctrine()->getManager();
+            $entityManager->persist($post);
+            $entityManager->flush();
+
+            // Flash messages are used to notify the user about the result of the
+            // actions. They are deleted automatically from the session as soon
+            // as they are accessed.
+            // See http://symfony.com/doc/current/book/controller.html#flash-messages
+            $this->addFlash('success', 'post.created_successfully');
+
+            if ($form->get('saveAndCreateNew')->isClicked()) {
+                return $this->redirectToRoute('admin_post_new');
+            }
+
+            return $this->redirectToRoute('admin_post_index');
+        }
+
+        return $this->render('admin/blog/new.html.twig', array(
+            'post' => $post,
+            'form' => $form->createView(),
+        ));
+    }
+
+    /**
```

Comment:
__[development][code_sniffer]__
* Doc comment for parameter "$post" missing

<!-- Comment by [SideCI](https://sideci.com) -->